Document `--cred-file` flag #582 (#583)
enonic/cli-enonic: docs/xp.adoc

|`ENONIC_CLI_HTTP_PROXY`
|URL of proxy server to use

|`ENONIC_CLI_CRED_FILE`

> |The path to the service account key file (in JSON format). This is only available for XP version 7.15 and later. Key file can be generated by Users application for System ID Provider users (aka Service Accounts). If specified, the flag `--auth` or `-a` will be ignored

Reviewer: ```suggestion
|Path to a service account key file (in JSON format). This flag will only work with XP 7.15 or later. A key file can be generated in the Users application for System ID Provider users (aka Service Accounts). If specified, the `--auth` (or `-a`) flag will be ignored.
```

     --skip-permissions      Flag to skips permissions when importing
     --dry                   Show the result without making actual changes.
     -a value, --auth value  Authentication token for basic authentication (user:password)

>      --cred-file value       The absolute path to the service account key file (in JSON format). This is only available for XP version 7.15 and later. Key file can be generated by Users application for System ID Provider users (aka Service Accounts) . If specified, the flag "--auth" or "-a" will be ignored

Reviewer: ```suggestion
     --cred-file value       Absolute path to a service account key file (in JSON format). This flag will only work with XP 7.15 or later. A key file can be generated in the Users application for System ID Provider users (aka Service Accounts). If specified, the `--auth` (or `-a`) flag will be ignored.
```

     --blob, -b              Also removes unused blobs
     --threshold, -t         Age of data to be removed in ISO-8601 duration format PnDTnHnMn.nS with days considered to be exactly 24 hours
     --auth value, -a value  Authentication token for basic authentication (user:password)

>      --cred-file value       The path to the service account key file (in JSON format). This is only available for XP version 7.15 and later. Key file can be generated by Users application for System ID Provider users (aka Service Accounts) . If specified, the flag "--auth" or "-a" will be ignored

Reviewer: ```suggestion
     --cred-file value       Path to a service account key file (in JSON format). This flag will only work with XP 7.15 or later. A key file can be generated in the Users application for System ID Provider users (aka Service Accounts). If specified, the `--auth` (or `-a`) flag will be ignored.
```
